package.json

```diff
@@ -5,29 +5,26 @@
   "main": "index.js",
   "repository": "https://github.com/artsy/metaphysics",
   "engines": {
-    "node": "8.4.x",
-    "npm": "5.4.x",
+    "node": "8.4.0",
+    "npm": "5.3.0",
     "yarn": "1.1.0"
   },
   "scripts": {
     "start": "yarn run dev",
     "verbose-dev": "DEBUG=verbose,info,error nf start --procfile Procfile.dev",
     "dev": "DEBUG=info,error nf start --procfile Procfile.dev -w",
-    "staging": "DEBUG=info,error nf start --procfile Procfile.dev --env .env.staging",
     "test": "yarn run lint && jest",
     "watch": "jest --watch",
     "lint": "eslint .",
     "fix": "eslint . --fix",
-    "promote": "heroku pipelines:promote -r staging",
     "dump-schema": "babel-node ./scripts/dump-schema.js",
+    "build": "yarn build:lib && yarn build:index && yarn build:fixtures",
```

Comment:
FYI I recently learned about the `-s` option for `yarn` which can be used to reduce the noise when invoking scripts from within scripts. E.g.

```
"build": "yarn -s build:lib && yarn -s build:index && yarn -s build:fixtures"
```

Reply:
Tried that but got the same output

Reply:
Hrmm. Possibly an older yarn version?